ingestion/src/metadata/ingestion/models/ometa_table_db.py

```diff
@@ -28,3 +29,9 @@ class OMetaDatabaseAndTable(BaseModel):
 class OMetaDatabaseAndModel(BaseModel):
     model: DbtModel
     database: Database
+
+
+class OMetaDatabaseAndTableAndLocation(BaseModel):
```

Comment:
can we add location as an optional field OMetaDatabaseAndModel such that we can continue use that class instead of adding another one.

Reply:
I'm on it.

Reply:
I have used inheritance instead of Optional. WDYT?

Reply:
why not add the location to OMetaDatabaseModel though? location is an optional component in the table anyway.
Reason for this is we are trying to keep the metadata_rest to have standard schema entities from generated code.
So if we keep the location to be optional in OMetaDatabaseAndModel all we need to all the sources we have is to optionally add location to existing entity